shoop/core/models/contacts.py

```diff
@@ -85,7 +85,10 @@ class Meta:
 
 class CompanyContact(Contact):
     members = models.ManyToManyField("Contact", related_name="company_memberships", blank=True)
-    vat_code = models.CharField(max_length=32, blank=True, verbose_name=_('VAT code'))
+    tax_number = models.CharField(
+        max_length=32, blank=True,
```

Comment:
Yes, I had a todo that "make all the tax number fields equal length". Didn't do that yet, as you can see..